superset/config.py

```diff
@@ -645,3 +645,6 @@ class CeleryConfig(object):
         )
 except ImportError:
     pass
+
+SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:postgres@localhost/superset'
```

Comment:
I think this should be in your local `superset_config.py`

Reply:
yes, my bad